options.html

```diff
@@ -30,11 +30,25 @@
 							<br />
 							<input type="checkbox" id="minihelp" name="minihelp">
 							Show mini help
+							<br />
+							<br />
+							Show popup with delay: 
+							<input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
 							</p>
 					</div>
 					<!-- <div id="keycon" class="tabcontent hide"> -->
 					<div id="keycon" class="tabContent">
 					<h1>Keyboard</h1>
+						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>
+						Immediately show only when key <select id="showOnKey" name="showOnKey"> 
+							<option value="">--</option> 
+							<option value="Alt">Alt</option> 
+							<option value="Ctrl">Ctrl</option> 
+							<option value="Alt+Ctrl">Alt+Ctrl</option> 
+						</select> is pressed.
+
+						<br />

```

Comment:
I guess the core is that it's an extra two characters that aren't needed.

A larger reason is that it's an XHTML idiom that feels out of place in pure
HTML.

I'm sure the requirements of my day job also color my review as well.

It's not really fair to hold everyone to some standard without even a
published style guide. :)

2016年9月9日(金) 12:57 Versus <notifications@github.com>:

> In options.html
> <https://github.com/melink14/rikaikun/pull/117#discussion_r78239437>:
>
> >  							</p>
> >  					</div>
> >  					<!-- <div id="keycon" class="tabcontent hide"> -->
> >  					<div id="keycon" class="tabContent">
> >  					<h1>Keyboard</h1>
> > +						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>
> > +						Immediately show only when key <select id="showOnKey" name="showOnKey">
> > +							<option value="">--</option>
> > +							<option value="Alt">Alt</option>
> > +							<option value="Ctrl">Ctrl</option>
> > +							<option value="Alt+Ctrl">Alt+Ctrl</option>
> > +						</select> is pressed.
> > +
> > +						<br />
>
> Why, by the way?
> [image: xkcd]
> <https://camo.githubusercontent.com/4665ea6462f4ea9998127a5e3676587946097224/687474703a2f2f696d67732e786b63642e636f6d2f636f6d6963732f282e706e67>
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/melink14/rikaikun/pull/117/files/c147b68a7cba9e32dd8009c4208a6932b3dbb478#r78239437>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHz5pRc8KP-qbaCztJPX1dph2JOf1e_ks5qoboygaJpZM4JD1OX>
> .
>